Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "IndirectArrays"
+uuid = "9b13fd28-a010-5f03-acff-a1bbcff69959"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+
+[compat]
+Compat = "0.53"
```

Comment:
Check CI, you'll see it's failing because of this line. Does this package actually use `Compat` anymore?